apps/backend/src/services/assetRegistrar/eam/api/createContainer.ts

```diff
@@ -151,18 +151,12 @@ export async function createContainer(shipmentId: number) {
     senderPhone ?? 'No value',
     instruments?.map((instrument) => instrument.shortCode) ?? ['No value']
   );
-
-  const response = await performApiRequest(request);
-
-  const regexFindEquipmentCode =
-    /<ns2:EQUIPMENTCODE>([0-9]*)<\/ns2:EQUIPMENTCODE>/;
-  const result = response.match(regexFindEquipmentCode);
-
-  if (!result || result.length < 2) {
+  const response = await performApiRequest('/equipment', request);
+  if (!response) {
```

Comment:
I believe this is always true if the response is OK. If it is not, the `performApiRequest` fn throws an error.